packages/api-plugin-fulfillment-method-pickup-store/src/getFulfillmentMethodsWithQuotesPickupStore.js

```diff
@@ -0,0 +1,75 @@
+import Logger from "@reactioncommerce/logger";
+import collectStoreDetails from "./util/collectStoreDetails.js";
+
+const packageName = "fulfillment-method-pickup-store";
+const fulfillmentTypeName = "pickup";
+const fulfillmentMethodName = "store";
+const logCtx = { name: "fulfillment-method-pickup-store", file: "getFulfillmentMethodsWithQuotesPickupStore" };
+
+/**
+ * @summary Returns a list of fulfillment method quotes based on the items in a fulfillment group.
+ * @param {Object} context - Context
+ * @param {Object} commonOrder - details about the purchase a user wants to make.
+ * @param {Array} [previousQueryResults] - an array of shipping rates and
+ * info about failed calls to the APIs of some shipping methods providers
+ * @returns {Array} - an array that contains two arrays: the first array will
+ * be an updated list of shipping rates, and the second will contain info for
+ * retrying this specific package if any errors occurred while retrieving the
+ * shipping rates.
+ * @private
+ */
+export default async function getFulfillmentMethodsWithQuotesPickupStore(context, commonOrder, previousQueryResults = []) {
+  const { collections: { Fulfillment } } = context;
+  const [rates = [], retrialTargets = []] = previousQueryResults;
+  const currentMethodInfo = { packageName };
+
+  if (retrialTargets.length > 0) {
+    const isNotAmongFailedRequests = retrialTargets.every((target) => target.packageName !== packageName);
+    if (isNotAmongFailedRequests) {
+      return previousQueryResults;
+    }
+  }
+
+  const pickupDocs = await Fulfillment.find({
```

Comment:
The naming `Docs` here is not really descriptive. It was previously used in the shipping-flat-rate plugin, but looks like it was picked because of naming negligence. I think it's better not to adopt this naming from the previous plugin but rather use something more unambiguous like `pickupFulfillmentType` because this is what the variable actually stores.

Reply:
Yes, I shall update it